src/Api/Modules/InternalModule.cs

```diff
@@ -0,0 +1,33 @@
+using Api.BackgroundServices;
+using Carter;
+using Core.Ods.Abstractions;
+using Core.Pds.Abstractions;
+
+namespace Api.Modules;
+
+public class InternalModule : CarterModule
```

Comment:
For completeness, should we add the endpoint for NDOP?

I have a task to add NDOP e2e tests which I'm hoping to work on soon, it would be great if the NDOP internal endpoint was there ready for this task 🙂

Reply:
Done. :)